src/managers/config.py

```diff
@@ -17,22 +18,26 @@
 logger = logging.getLogger(__name__)
 
 
-DEFAULT_PROPERTIES = """
-opensearch.ssl.verificationMode: full
-opensearch.requestHeadersWhitelist: [authorization, securitytenant]
-opensearch_security.multitenancy.enabled: true
-opensearch_security.multitenancy.tenants.preferred: [Private, Global]
-opensearch_security.readonly_mode.roles: [kibana_read_only]
-opensearch_security.cookie.secure: true
-"""
+DEFAULT_PROPERTIES = {
+    "opensearch.requestHeadersWhitelist": ["authorization", "securitytenant"],
+    "opensearch_security.multitenancy.enabled": True,
+    "opensearch_security.multitenancy.tenants.preferred": ["Private", "Global"],
+    "opensearch_security.readonly_mode.roles": ["kibana_read_only"],
+    "server.ssl.enabled": False,
+    "opensearch_security.cookie.secure": False,
+    "server.ssl.supportedProtocols": ["TLSv1.2"],  # In tandem with current OpenSearch
```

Comment:
In OpenSearch we are unfortunately stuck with TLSv1.2 because of the CA rotation topic. But in the dashboards we are not, I believe we can delete this, which would default to `['TLSv1.1', 'TLSv1.2', and 'TLSv1.3']`